src/shared/components/Header/index.tsx

```diff
@@ -2,7 +2,9 @@ import * as React from 'react';
 import { Menu, Dropdown, Icon, Button, Popover } from 'antd';
 import './Header.less';
 import Copy from '../Copy';
-
+import { NexusClient } from '@bbp/nexus-sdk';
```

Comment:
You cannot make use of the SDK (or any other side effect libraries) in a component. As a global strategy, this is why we have `components` and `containers`